### DISCUSSION TITLE
Ori and the Blind Forest

### DISCUSSION THREAD

**Discussion by alikecheaziz** [skyline-emu/skyline#759] [Game Requests]
ori and the blind forest

**Comment by marcoluc97** [comment]
Just a question for Skyline @PixelyIon team, but this can help?
`unimplemented engine: 0xAB args: 0x3`
`unimplemented engine: 0xA4 args: 0x0`
`unimplemented engine: 0x221 args: 0x3F`
`unimplemented engine: 0x54A args: 0x0`
`unimplemented engine: 0x982 args: 0x2`
`unimplemented engine: 0x1DF args: 0x1000000`
`unimplemented engine: 0x85 args: 0x3000000`
`unimplemented engine: 0x582 args: 0x4`
`unimplemented engine: 0x583 args: 0x0`
`unimplemented engine: 0xC4 args: 0x503`
`Unimplemented SVC 0x32 (SVC: 0x32)`

Thanks in advance ☺

**Reply by PixelyIon** [reply]
We're already aware of this being the reason that all Unity games don't boot. It's just that we're busy with various tasks and haven't gotten to Unity games yet.

**Comment by Giiithubb** [comment]
I want to play it too
